roles/openshift_master_upgrade/tasks/systemd_units.yml

```diff
@@ -0,0 +1,182 @@
+---
+# input variables
+# - openshift.node.node_image
+# - openshift_image_tag
+# - openshift.common.is_containerized
+# - openshift.node.ovs_image
+# - openshift.common.use_openshift_sdn
+# - openshift.common.service_type
+# - openshift.node.debug_level
+# - openshift.common.config_base
+# - openshift.common.http_proxy
+# - openshift.common.portal_net
+# - openshift.common
+# - openshift.common.http_proxy
+# notify:
+# - restart openvswitch
+# - restart node
+
+# This file is included both in the openshift_master role and in the upgrade
+# playbooks.  For that reason the ha_svc variables are use set_fact instead of
+# the vars directory on the role.
+
+- name: Init HA Service Info
+  set_fact:
+    containerized_svc_dir: "/usr/lib/systemd/system"
+    ha_svc_template_path: "native-cluster"
```

Comment:
Any reason to not just define the vars in a common parent directory for both roles? There are some times when set_fact is needed, but it does incur a cost that can be avoided if defined in another way.

Alternatively, these variables could be set at the block level as well.